config/locales/registration_transfers.en.yml

```
@@ -0,0 +1,37 @@
+en:
+  registration_transfers:
+    new:
+      title: "Transfer a registration to a different user account"
+      heading: "Transfer registration %{reg_identifier} to a different user account"
+      paragraph_1: "This registration currently belongs to %{email}."
+      paragraph_2: "If you know that a different user should own %{reg_identifier}, you can transfer it to a different account."
+      paragraph_3: "Once the registration is transferred, %{email} will no longer be able to view or manage it."
+      paragraph_4: "Any other registrations owned by %{email} will not be affected."
+      registration_info:
+        heading: "About this registration"
+        labels:
+          account_email: "Current user account"
+          reg_identifier: "Registration number"
+          company_name: "Company name"
+      subheading_1: "Transfer the registration"
+      email_label: "Email address"
+      confirm_email_label: "Confirm email address"
+      button: "Transfer account"
```

As we are transferring just a single registration I think have the submit button labelled **Transfer account** would be confusing. I think simply labelling it **Transfer** is sufficient in the context of all the other information we are showing.

```suggestion
      button: "Transfer"
```

---

🤦‍♀️  You're totally right! That's definitely inaccurate.